addons/website/static/tests/tours/media_iframe_video.js

```diff
@@ -0,0 +1,65 @@
+/** @odoo-module */
+import wTourUtils from "website.tour_utils";
+
+wTourUtils.registerWebsitePreviewTour(
+    "website_media_iframe_video",
+    {
+        test: true,
+        url: "/",
+        edition: true,
+    },
+    [
+        wTourUtils.dragNDrop({
+            id: "s_text_image",
+            name: "Text - Image",
+        }),
+        {
+            content: "Select the image",
+            trigger: "iframe #wrap .s_text_image img",
+            run: "click",
+        },
+        {
+            content: "Open image link options",
+            trigger: "[data-name='media_link_opt']",
+            run: "click",
+        },
+        {
+            content: "Enter the url",
+            trigger: "input[placeholder='www.example.com']",
+            run: "text odoo.com",
+        },
+        {
+            content: "Click on replace media",
+            trigger: "[data-replace-media='true']",
+            run: "click",
+        },
+        {
+            content: "Click on video button",
+            trigger: "a:contains('Videos')",
+            run: "click",
+        },
+        {
+            content: "Enter video link",
+            trigger: "#o_video_text",
+            run: "text https://youtu.be/nbso3NVz3p8",
+        },
+        {
+            content: "Check video is preview",
+            trigger: ".o_video_dialog_iframe",
```

Comment:
Sorry, but I'm only seeing it now => In 16.0, "click" is the default action.. So add `run: () => {}, // This is a check.` here and remove all the `run: "click",` from the test